cluster-agent/README.md

```diff
@@ -8,7 +8,44 @@ This chart deploys AppDynamics ClusterAgent for monitoring Kubenetes and OpenShi
 * Kubernetes 1.7+ OpenShift 3.7+
 * Helm and tiller installed in the cluster with the appropriate RBAC settings
 * Kubernetes [Metrics server](https://hub.helm.sh/charts/stable/metrics-server) installed
-* An AppDynamics account. AppDynamics offers a [free trial](https://www.appdynamics.com/free-trial/)
+* An AppDynamics account.
+
+### Create a secret
+Secure and recommended approach to provide sensitive data like passwords is to generate a secret containing the sensitive data and providing the pre-created secret name during chart deployment.
+
+You can provide an existing secret containing `api-user` if applicable `controller-key` and `event-key`.
+The existing secret can be provided as a value to `existingSecret`.
+
+If creating a secret, it is recommended to name the secret as `release-name`-`secret-name`. For example:  MyRelease-MySecret
+
+This allows the secrets to be affiliated with the release, but not directly included with the release.
+Use below command to create a secret:
+```
+kubectl create secret generic my-release-<secret_name> \
+  --from-literal='api-user=XXXX' \
+  --from-literal='controller-key=XXXX' \
+  --from-literal='event-key=XXXX' \
+  --namespace namespace_name
+```
+
+### Image Security Policies
+
+If the cluster has image security policies enforced, cluster-agent docker images should be added to it.
+
+```
+apiVersion: securityenforcement.admission.cloud.ibm.com/v1beta1
+kind: ClusterImagePolicy
+metadata:
+name: ibmcloud-default-cluster-image-policy
+spec:
+ repositories:
+   - name: docker.io/appdynamics/cluster-agent:*
+```
+
+### Pod Security Policies
```

Comment:
I noticed the container runs as root.
ICP recommends to run the container as a non-root user. This is not a hard requirement for our first round.
It will become a mandate if this chart intends to become a cloud pak. Cloud paks are required to run containers as non-root users and have to specify a security context in deployment.
For our initial take, I have added documentation for user to create a namespace with less restrictive PSP that allows containers to run as root.